samples/bluetooth/scanner_rssi/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  description: TBD
+  name: TBD
+tests:
+  sample.bluetooth.scanner.rssi:
+    platform_allow: nrf5340dk_nrf5340_cpuapp, nrf5340dk_nrf5340_cpuappns, nrf5340pdk_nrf5340_cpuapp, nrf5340pdk_nrf5340_cpuappns, nrf52dk_nrf52832, nrf52840dk_nrf52840, nrf52833dk_nrf52820, nrf52833dk_nrf52833
```

Comment:
I don not really know what this file is for or what system uses it. Should I be listing all those supported platforms here too, or is the readme sufficient ? I did test the build for all of them, but only ran on 52840dk and 5340pdk